### PR TITLE
Removing Contains Reviewer Comment flag

### LINKED ISSUE #7516: Remove "Contains Reviewer Comment" flag
When a reviewer posts a comment for other reviewers (not for the developer) in reviewer tools, it's shown in the activity log for reviewers, but not developers. In addition, a flag is stored and displayed in the queue for the add-on, indicating that it contains a reviewer comment.

I'm wondering how useful the flag is - essentially it will only be shown in the queues the add-on appears in. It's stored on the version and is a bit wasteful. Can it be dropped or is it still useful?

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #7516

### CODE REVIEW

**Comment by eviljeff** [comment]
there's more to remove here - if you do a search for `has_reviewer_comment` you'll see all the matches (including the fixtures)

File: src/olympia/reviewers/utils.py [hunk 1]
```diff
@@ -622,7 +622,6 @@ def reviewer_reply(self):
 
     def process_comment(self):
         if self.version:
-            kw = {'has_reviewer_comment': True}
```
Comment (championshuttler) [inline src/olympia/reviewers/utils.py]: @eviljeff can you please guide me what to do here i think it will fail after removing from here

Comment (eviljeff) [reply src/olympia/reviewers/utils.py]: You can collapse it down to a single if and `.update()` call.
```python
update_reviewed = (
    self.version and
    self.version.channel ==  amo.RELEASE_CHANNEL_UNLISTED and
    not self.version.reviewed)
if update_reviewed:
    self.version.update(reviewed=datetime.now())
```

File: src/olympia/reviewers/utils.py [hunk 2]
```diff
@@ -621,13 +621,13 @@ def reviewer_reply(self):
                        detail_kwargs={'reviewtype': self.review_type})
 
     def process_comment(self):
-        if self.version:
-            kw = {'has_reviewer_comment': True}
-            if (self.version.channel == amo.RELEASE_CHANNEL_UNLISTED and
-                    not self.version.reviewed):
-                kw['reviewed'] = datetime.now()
-            self.version.update(**kw)
-        self.log_action(amo.LOG.COMMENT_VERSION)
```
Comment (eviljeff) [inline src/olympia/reviewers/utils.py]: you still need this line (that's why you have some test fails)

**Comment by championshuttler** [comment]
Should I create another issue and PR for it?

**Comment by championshuttler** [comment]
@eviljeff  raised another PR for it https://github.com/mozilla/addons-server/pull/9145

**Comment by eviljeff** [comment]
@championshuttler actually it's best to add it in another commit to this branch/pull-request so it all lands in one go.

**Comment by championshuttler** [comment]
Oops ok :man_facepalming:

**Comment by championshuttler** [comment]
Hi @eviljeff sorry for delay :man_facepalming:  added a file here :D

**Pull request merged**